zulipterminal/ui_tools/views.py

```diff
@@ -1006,9 +1006,13 @@ def __init__(self, controller: Any, title: str) -> None:
         super().__init__(controller, widgets, 'HELP', popup_width, title)
 
 
+PopUpConfirmationViewLocation = Literal['top-left', 'center']
```

Comment:
I think this comment should go before the previous one, as there is nowhere near enough space in the top-left for showing the download information :)

Reply:
You mean commit?